randovania/games/samus_returns/presets/multiworld-starter-preset.rdvpreset

```diff
@@ -0,0 +1,285 @@
+{
+    "schema_version": 82,
+    "base_preset_uuid": null,
+    "name": "Multiworld Starter Preset",
+    "uuid": "7b724691-d556-410f-aca1-8b1d83ca77b9",
+    "description": "A modified version of the Starter Preset that is more fine tuned for Multiworld.",
+    "game": "samus_returns",
+    "configuration": {
+        "trick_level": {
+            "minimal_logic": false,
+            "specific_levels": {}
+        },
+        "starting_location": [
```

Comment:
are these custom locations supposed to be there?

Reply:
Yes, I hand picked some starts that wouldn't be locked by too much so there was some variance.